core-bundle/contao/templates/twig/component/_list.html.twig

```diff
@@ -94,7 +94,11 @@
     {% if randomize_order or add_pagination or limit_elements %}
         {% block list_script %}
             {% add "list_script" to body %}
-                <script>
+                {% set script_attributes = attrs(script_attributes|default)
+                    .setIfExists('nonce', contao_csp_nonce('script-src'))
+                %}
+
+                <script{{ script_attributes }}>
```

Comment:
```suggestion
                <script{{ attrs(script_attributes|default).setIfExists('nonce', contao_csp_nonce('script-src')) }}>
```

I think this can be shortened this way.

Reply:
Yes, but I was adhering to the style of the other HTML attributes in all our Twig templates.